dev-python/pyilmbase/pyilmbase-2.2.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 cmake-utils
```

Comment:
You can sort them lexically, they don't collide.

Reply:
Ah. Sorry. I was told ages ago that sometimes certain ones had to go last.